pkg/benchmarkjunit/main.go

```diff
@@ -66,11 +69,23 @@ func run(opts *options, args []string) {
 	testCmd := exec.Command(opts.goBinaryPath, testArgs...)
 
 	logrus.Infof("Running command %q...", append([]string{opts.goBinaryPath}, testArgs...))
-	testOutput, testErr := testCmd.CombinedOutput()
+	var testOutput []byte
+	var testErr error
+	if opts.logFile == "-" {
+		// Stream command output to stdout.
+		var buf bytes.Buffer
+		writer := io.MultiWriter(os.Stdout, &buf)
+		testCmd.Stdout = writer
+		testCmd.Stderr = writer
+		testErr = testCmd.Run()
+		testOutput = buf.Bytes()
+	} else {
+		testOutput, testErr = testCmd.CombinedOutput()
+	}
 	if testErr != nil {
 		logrus.WithError(testErr).Error("Error(s) executing benchmarks.")
 	}
-	if len(opts.logFile) > 0 {
+	if len(opts.logFile) > 0 && opts.logFile != "-" {
```

Comment:
Nit: maybe move this to a switch, might also make it more flexible if more options are added later on?
```
case opts.logFile == "-":
		// Stream command output to stdout.
		var buf bytes.Buffer
		writer := io.MultiWriter(os.Stdout, &buf)
		testCmd.Stdout = writer
		testCmd.Stderr = writer
		testErr = testCmd.Run()
		testOutput = buf.Bytes()
case len(opts.logFile) > 0:
       testOutput, testErr = testCmd.CombinedOutput()
       if err := ioutil.WriteFile(opts.logFile, testOutput, 0666); err != nil {
			logrus.WithError(err).Fatalf("Failed to write to log file %q.", opts.logFile)
       }
```

Reply:
nvm, that doesn't quite work with `if testErr != nil`